.github/workflows/build-images-action.yml

```diff
@@ -0,0 +1,22 @@
+name: build-images-action
+on: 
+  push:
+    branches: [ main ]
+jobs:
+  build:
+    name: Build container images
+    runs-on: ubuntu-latest
+    steps:
+      - name: build mariabd image
+        uses: toptal/jenkins-job-trigger-action@master
```

Comment:
We will also need to have it use a digest pinned version, with human readable version in a comment after. See other workflow files.

Reply:
Sorry I'm not sure I completely get your idea. Is replacing @master with @release-tag enough, or do you mean something else?

Reply:
Format is `uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0` for example. 

Repo owner could push breaking or even malicious code to 'master`, or retag a release, which we don't want. So we pin with digests.

Reply:
For Github Actions, the digest is Git commit SHA, and the tag is the Git release tag. So you find the digest by checking the SHA of the tag and vice versa.

Reply:
Thanks for the explanation. I'll add it.